flashcards/backend/docs/manage.rst

```diff
@@ -0,0 +1,7 @@
+manage module
```

Comment:
Almost missed it. Remove this .rst file. We do not need to document django manage module. They habe their own documentation 😅